ibm/service/cis/resource_ibm_cis_mtls.go

```diff
@@ -0,0 +1,217 @@
+// Copyright IBM Corp. 2017, 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	cisMtlsID            = "mtls_id"
+	cisMtlsCert          = "cert_mtls"
+	cisMtlsCertName      = "cert_name"
+	cisMtlsHostName      = "host_name"
+	cisMtlsCertCreatedAt = "created_at"
+	cisMtlsCertUpdatedAt = "updated_at"
+	cisMtlsCertExpireOn  = "expires_on"
+)
+
+func ResourceIBMCISMtls() *schema.Resource {
+	return &schema.Resource{
+		Create:   ResourceIBMCISMtlsCreate,
```

Comment:
Move to the new feature context (CreateContext , ReadContext, UpdateContext, DeleteContext)
Eg https://github.com/IBM-Cloud/terraform-provider-ibm/blob/bc578cb2386b76d4a6720a0aaec6ddcf314fd827/ibm/service/contextbasedrestrictions/resource_ibm_cbr_rule.go#L23